src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrScannerTest.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Darc.Tests.VirtualMonoRepo;
+using Microsoft.DotNet.DarcLib.Helpers;
+using Microsoft.DotNet.DarcLib.Models.VirtualMonoRepo;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Darc.Tests.VirtualMonoRepo;
+
+[TestFixture]
+public class VmrScannerTest : VmrTestsBase
+{
+    [Test]
+    public async Task VmrScannerFindsNothingTest()
+    {
+        await InitializeRepoAtLastCommit(Constants.ProductRepoName, ProductRepoPath);
+
+        var list = await CallDarcScan();
+        Assert.AreEqual(0, list.Count);
```

Comment:
I'd remove this test and maybe make this check a part of another as we do this in all other tests and the tests take some time to run.

Reply:
I could also make them all one test

Reply:
It might make sense as they are more of the E2E nature

Reply:
I put everything in one test